communication/README.md

```diff
@@ -36,6 +36,16 @@ devoted to the Kubernetes project. Please check the guidelines and any relevant
 chat/conversation history before posting. Spam and sales pitches are not tolerated
 on these platforms.
 
+### Appropriate Content for Community Resources
+
+All communications properties are under the [Kubernetes code of conduct]. 
+Additionally, these resources are for the contributors and users of Kubernetes, commercial usage of these properties is heavily moderated.
+
+Examples of inappropriate content:
```

Comment:
Could we lead with similar short section of "Examples of appropriate content" for a more positive, community tone?